tests/ZfrCorsTest/Factory/CorsOptionsFactoryTest.php

```diff
@@ -39,4 +39,13 @@ public function testCanCreateOptions()
 
         $this->assertInstanceOf('ZfrCors\Options\CorsOptions', $options);
     }
+
+    public function testCanCreateOptionsWithServiceManagerV2()
```

Comment:
If this and the other "v2" tests are what are failing, my point above is that they're unnecessary if you're using invokables without implementing `FactoryInterface`.